default/stringtables/en.txt

```diff
@@ -4705,6 +4705,29 @@ Important sections include [[encyclopedia CATEGORY_GAME_CONCEPTS]] and the recen
 
 '''
 
+SITREP_DISMISS_HIDE_TITLE
+Dismissing and Hiding SitReps
+
+SITREP_DISMISS_HIDE_SHORT_DESC
+Dismissing a specific SitRep and hiding SitRep types.
+
+SITREP_DISMISS_HIDE_TEXT
+'''There are two options for ignoring Situation Reports (SitReps): Dismissing and Hiding.
+
+Dismissing a SitRep will keep that specific SitRep for displaying.
```

Comment:
This does not clarify anything for me. A sitrep is generated on and describes an event on a specific turn... how does one "dismiss" a "specific sitrep" for several turns? Does that mean if I hide it on turn 10, then on turn 13 I go back in the sitrep to look at turn 10, a single entry will be hidden? I don't see why anyone would want to do that, so I assume that's not what it means. So then I supposed something else is meant by "specific sitrep", such as a particular type of message, like "enemy fleet arrived"... but then what is a "group of sitreps" if not a particular type of message?